packages/config/src/ConfigSchema.ts

```diff
@@ -28,13 +35,30 @@ export class ConfigSchema {
      */
     private static readonly SCHEMA_VERSION = 3;
 
+    /**
+     * Pretty explanation of the schema objects
+     * @readonly
+     * @memberof ConfigSchema
+     */
+    private static readonly explainSchemaSummary: IExplanationMap = {
+        $schema: "URL",
+        $version: "Version",
+        properties: {
+            defaults: "Default Definitions",
+            explainedParentKey: "Properties",
+            ignoredKeys: null
+        },
+        explainedParentKey: "Schema",
+        ignoredKeys: null
+    };
```

Comment:
This may not be super necessary.
Mostly for a nice `<app>.log` output. 
Another method would be to use `util.inspect`